sdks/go/node/api/handler/handler.go

```diff
@@ -41,6 +41,9 @@ func (hdlr _handler) ServeHTTP(
 	httpResp http.ResponseWriter,
 	httpReq *http.Request,
 ) {
+	// limit req size to 1 Gb
+	httpReq.Body = http.MaxBytesReader(httpResp, httpReq.Body, 1000000000)
```

Comment:
@apexskier req size limiting added here

Reply:
This limits it on the remote only, right? I'm more concerned about memory explosions on the users local computer.

Reply:
I missunderstood your ask; I was thinking because it's in the clients hands they would check that if they cared but I guess being an SDK it could be nice to do it for them. Should be good to go now https://github.com/opctl/opctl/pull/886/files#diff-77477245ac34983a608813751c412801f40e453bcce7a691930704d74bcf61dbR13

Reply:
Also, I reduced max size down to 40mb. I think until we have a more sophisticated implementation it's reasonable for us to be overly restrictive. The case I know of where folks were sending up to 2Gb can be handled by the user.